changelog.d/+testing-guide.internal.md

```diff
@@ -0,0 +1 @@
+Improve testing guide.
```

Comment:
This actually isn't internal - should be relevant to external contributors as well

Reply:
I think the internal/external border should pass between mirrord developers and mirrord users, and not between the core team and the rest of the world. So I would also put it in internal.

Reply:
Fair enough